README.md

```diff
@@ -1 +1,75 @@
+- Sexist, racist, homophobic, transphobic, ableist or otherwise discriminatory jokes and language.
+- Verbal comments that reinforce social structures of domination related to gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, age, religion
+- Posting or displaying sexually explicit or violent material.
+- Posting or threatening to post other people's personally identifying information ("doxing").
+- Personal insults, particularly those related to gender, sexual orientation, race, religion, or disability.
+- Inappropriate or unwelcome photography or recording.
+- Inappropriate physical contact. You should have someone's consent before touching them.
+- Unwelcome sexual attention. This includes, sexualized comments or jokes; inappropriate touching, groping, and unwelcome sexual advances.
+- Deliberate intimidation, stalking or following (online or in person).
+- Sustained disruption of community events, including talks and presentations.
+- Advocating for, or encouraging, any of the above behavior.
+
+## Weapons Policy
+
+No weapons will be allowed at Front-End Party events, community spaces, or in other spaces covered by the scope of this Code of Conduct. Weapons include but are not limited to guns, explosives (including fireworks), and large knives such as those used for hunting or display, as well as any other item used for the purpose of causing injury or harm to others. Anyone seen in possession of one of these items will be asked to leave immediately, and will only be allowed to return without the weapon. Community members are further expected to comply with all state and local laws on this matter.
+
+## Consequences of Unacceptable Behavior
+
+Unacceptable behavior from any community member, including sponsors and those with decision-making authority, will not be tolerated.
+
+Anyone asked to stop unacceptable behavior is expected to comply immediately.
+
+If any community member, participant, or attendee engages in unacceptable behavior, the community organizers may take any action they deem appropriate, up to and including a temporary ban or permanent expulsion from the community and/or event without warning (and without refund in the case of a paid event).
+
+## Reporting 
+
+If you are subject to or witness unacceptable behavior, feel unsafe or unwelcome, or have any other concerns, please notify a community organizer as soon as possible by either directly contacting the organizers or filing an anonymous report:
+
+### Direct Reporting
+
+Directly contact one of the organizers in person or through Meetup’s messaging system:
+
+- Cameron Sampson https://www.meetup.com/FrontEndParty/members/52557152/ 
+- Steven Achord https://www.meetup.com/FrontEndParty/members/185213572/ 
+- Sia Karamalegos https://www.meetup.com/FrontEndParty/members/98819682/ 
+- Matt Shwery https://www.meetup.com/FrontEndParty/members/21058911/ 
```

Comment:
Perhaps rather than keeping the community organizers listed here we can point to a Meetup link that displays all of them?